enterprise/ha/src/test/java/org/neo4j/kernel/ha/lock/SlaveLocksClientTest.java

```diff
@@ -447,4 +477,11 @@ private SlaveLocksClient stoppedClient()
         client.stop();
         return client;
     }
+
+    private SlaveLocksClient closedClient()
+    {
+        client.close();
+        client.acquireShared( NODE, 1 ); // trigger new lock session initialization
```

Comment:
Ooh, it does. So tests that I've added use `@Test` with expected exception. And this expected exception was thrown, just not from the place I want. Will swap these two lines and change tests.